### PR TITLE
Show sword forge bounty on 'Unlikely Victory' fights

### PR DESCRIPTION
**Pull request by kcper**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?
N/A
### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Highlighting and bounty icon on fights that are unlikely to be won. 
Icon source: https://www.pngwing.com/en/free-png-hkwrx with a filter added to it.

### CODE REVIEW

**Review by seiyria — APPROVED** [review]

**Comment by seiyria** [comment]
Looks good but I'll hold off on merging until the counterpart in solidity works.

**Review by raymond-h — CHANGES_REQUESTED** [review]
Weapons aren't forged automatically on win - instead the user is given a "ticket" that they can spend to mint a weapon instead of SKILL.

**Comment by kcper** [comment]
Summary:
Forge with ticket button:
image
Ticket amount picker if user has more than 1 ticket available:
image
Button if there are no tickets available, left disabled instead of hidden to show that there is this new ticket thing:
image
Unlikely fight:
image

File: frontend/src/store.ts [hunk 1]
```diff
@@ -953,6 +963,25 @@ export function createStore(web3: Web3) {
         ]);
       },
 
+      async spendTicketN({ state, dispatch }, {num}) {
+        console.log(num);
```
Comment (seiyria) [inline frontend/src/store.ts:967]: gotta remove the console.logs

File: frontend/src/views/Blacksmith.vue [hunk 2]
```diff
@@ -257,6 +283,28 @@ export default {
         this.$dialog.notify.error('Could not forge sword: insuffucient funds or transaction denied.');
       }
     },
+
+    async onTicketForge() {
+      console.log(this.forgeTicketCount);
```
Comment (seiyria) [inline frontend/src/views/Blacksmith.vue:288]: log

**Review by seiyria — APPROVED** [review]
overall UI looks good - I dig the idea and implementation. Just some console.logs but that's not going to hold this up.

**Review by raymond-h — APPROVED** [review]

**Comment by poshdan** [comment]
This should be delayed until we have 100% safe randoms to plug into fights, else it will be abused.
Given that our best random-safety measure requires 2tx I'm not sure this feature is viable for us.

**Pull request closed without merging**